packages/react-sdk/src/useMyself.ts

```diff
@@ -0,0 +1,51 @@
+import type { Member, Myself, SyncAgent } from '@river-build/sdk'
+import { useMemo } from 'react'
+import { type ActionConfig, useAction } from './internals/useAction'
+import { type ObservableConfig, useObservable } from './useObservable'
+import { useSyncAgent } from './useSyncAgent'
+import { getRoom } from './utils'
+
+const getMyself = (sync: SyncAgent, streamId: string) => getRoom(sync, streamId).members.myself
```

Comment:
can we call this useMyMember instead of useMyself? useMySelf sounds like it would return everything about myself...